app/views/loans/_loan_list.html.erb

```diff
@@ -43,13 +43,12 @@
           <% end %>
         </td>
 
-        <td><%= l.renewal_count %></td>
```

Comment:
removing the <td> around this causes the formatting of the table to no longer be square on my browser
![screenshot 2015-07-17 11 07 30](https://cloud.githubusercontent.com/assets/2886826/8753925/6755b4de-2c74-11e5-8953-4727d40be9cb.png)
.